docker/dockerfile.merlin

```diff
@@ -265,8 +265,9 @@ RUN apt update -y --fix-missing && \
 
 RUN ln -s /usr/bin/python3 /usr/bin/python
 
-# CMake
-COPY --chown=1000:1000 --from=build /usr/local/bin/cmake /usr/local/bin/cmake
+# Install cmake (used by downstream images)
```

Comment:
There are a couple of lines below this where we're copying cmake from `lib` and `bin`. Might be able to remove those if this is a replacement for those.

Reply:
It's actually supposed to be a replacement for installing CMake in the downstream images, which was happening over the top of those copies. I think we can remove them though

Reply:
Added a commit to address this